iml-services/iml-device/src/main.rs

```diff
@@ -106,29 +245,22 @@ async fn main() -> Result<(), ImlDeviceError> {
 
     tokio::spawn(server);
 
-    let mut s = consume_data::<Device>("rust_agent_device_rx");
+    let mut s = consume_data::<(Device, HashSet<Mount>)>("rust_agent_device_rx");
 
-    while let Some((f, d)) = s.try_next().await? {
-        let mut cache = cache2.lock().await;
+    // Django's artefact:
+    let lustreclientmount_ct_id = djct::table
+        .select(djct::id)
+        .filter(djct::model.eq("lustreclientmount"))
+        .first_async::<i32>(&pool)
+        .await
+        .ok();
```

Comment:
```suggestion
    let lustreclientmount_ct_id = djct::table
        .select(djct::id)
        .filter(djct::model.eq("lustreclientmount"))
        .first_async::<i32>(&pool)
        .await
        .optional()?;
```

This differentiates between a db error and the row not existing, instead of treating everything as an error.